rtb/analisi-requisiti/src/descrizione.tex

```diff
@@ -1,3 +1,46 @@
 \section{Descrizione }
 \subsection{Obiettivi del prodotto}
+
+Il prodotto, la cui idea nasce nel contesto del settore della ristorazione,
+ consiste in una responsive web app e si pone molteplici obiettivi:
+\begin{itemize}
+    \item Offire ai clienti la possibilità di effettuare la prenotazione presso un ristorante
```

Comment:
typo "offrire"
Secondo me cambiare frase in "...la prenotazione e l'ordinazione presso un ristorante"